### ISSUE TITLE
Uncaught TypeError: t is not a function - fields that have a character count/word

### ISSUE THREAD

**Issue by anchovy** [verbb/formie#1919]
### Describe the bug

This seems to be simular to the issue #1740 - but it's a max-characters limit that triggering it this time.

When we've got a character limit - on the first character entered we get the console error:
```
TypeError: t is not a function
    at text-limit.js?v=1717970014:1:25277
```

If I then change it to a 'Words' limit - on the first character entered we get the console error:
```
text-limit.js?v=1717970014:1 Uncaught TypeError: t is not a function
    at text-limit.js?v=1717970014:1:25647
```

### Steps to reproduce

1. Add a new text field and limit the field to 4 characters or words
2. Load the form and enter a  4 character and view the console

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: Yes

### Craft CMS version

Craft Pro 3.9.13

### Plugin version

1.6.43

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-3 as 1.6.43"`.

**Comment by anchovy** [comment]
Thanks @engram-design 
What's the eta for this release - this code is out on live.

**Comment by engram-design** [comment]
@anchovy Usually within a few days, probably mid-week.

**Comment by engram-design** [comment]
Fixed in [1.6.44](https://github.com/verbb/formie/releases/tag/1.6.44)
